server/remote/common.go

```diff
@@ -44,3 +80,12 @@ func (dt DateTime) Time() time.Time {
 	}
 	return t
 }
+
+func safeTimeZone(timeZone string) string {
```

Comment:
Not sure on the name here. This function takes a time zone, and converts it to a Go-compatible one if it is not compatible.

Reply:
well, why not call it `GoTimeZone` (and `MicrosoftTimeZone`)? Also I'd make it public, if nothing else for documentation purposes so they appear in GoDoc.